bindings/wasm/bindings.cpp

```diff
@@ -72,6 +73,13 @@ Manifold Transform(Manifold& manifold, std::vector<float>& mat) {
   return manifold.Transform(matrix);
 }
 
+Manifold Warp(Manifold& manifold, void* funcPtr) {
```

Comment:
I think my error message is related to this answer: https://stackoverflow.com/a/27364643

Reply:
Okay, not sure why I didn't just try what the suggested first...